examples/prestapyt_xml.py

```diff
@@ -8,17 +8,17 @@
 
 prestashop.get('')
 
-print "GET"
-print ElementTree.tostring(prestashop.get('addresses'))
-print ElementTree.tostring(prestashop.get('addresses', resource_id=1))
-print ElementTree.tostring(prestashop.get('addresses/1'))
-print ElementTree.tostring(prestashop.get('addresses', options={'limit': 1}))
-print ElementTree.tostring(prestashop.get('stock_movement_reasons'))
+print("GET")
```

Comment:
Better to create a new file with `_python3` suffix or similar and let this one for those using Python 2.

Reply:
For example files only or for the whole library ?